scripts/enum/magic.lua

```diff
@@ -64,22 +64,6 @@ xi.magic.element =
 }
 xi.magic.ele = xi.magic.element
 
------------------------------------
--- Day to Element Mapping
------------------------------------
-
-xi.magic.dayElement =
-{
-    [xi.day.FIRESDAY]     = xi.magic.element.FIRE,
-    [xi.day.ICEDAY]       = xi.magic.element.ICE,
-    [xi.day.WINDSDAY]     = xi.magic.element.WIND,
-    [xi.day.EARTHSDAY]    = xi.magic.element.EARTH,
-    [xi.day.LIGHTNINGDAY] = xi.magic.element.THUNDER,
-    [xi.day.WATERSDAY]    = xi.magic.element.WATER,
-    [xi.day.LIGHTSDAY]    = xi.magic.element.LIGHT,
-    [xi.day.DARKSDAY]     = xi.magic.element.DARK,
-}
-
```

Comment:
Why was this moved to magic.lua? I'm guessing it's only used there

Reply:
Because every table in it is attached to xi.magic

Reply:
Oh, and that one depends on other enums